site/content/entry/yuzu-progress-report-aug-2023/index.md

```diff
@@ -0,0 +1,542 @@
+{{< gh-hovercard "11375" "Regression solved, game stable, case closed." >}}
+
+{{< imgs
+	"./ai.png| Elementary, my dear user (AI: THE SOMNIUM FILES)"
+  >}}
+
+And finally, to close this section with what is more of a build process change than a graphics code change, [vonchenplus](https://github.com/vonchenplus) added {{< gh-hovercard "11302" "MoltenVK as an external dependency" >}} to yuzu’s CMake settings, allowing Apple users to automatically get the latest version of this translation library when building yuzu on their Vulkan-starved machines. (AGXV, anyone?)
+
+{{< imgs
+	"./m1.png| Let’s begin with simple stuff (DRAGON QUEST III The Seeds of Salvation)"
+  >}}
+
+Properly keeping up with MoltenVK updates allows macOS builds to run some basic games, but there’s still much to do to get the forbidden fruit company's devices up to speed.
+
+## OpenGL-specific improvements, which also improve Vulkan
+
+That’s right, Mesa and NVIDIA Fermi/Kepler users, it’s your turn to get some love.
+[Epicboy](https://github.com/ameerj) is back with some great changes for the ~~old~~classic API that started it all, OpenGL.
+
+First on the list is a major overhaul for GPU ASTC decoding, for both OpenGL and Vulkan APIs.
+{{< gh-hovercard "11149" "Code optimizations" >}} in several areas of the compute shader-based decoder improved Vulkan ASTC decoding performance by up to 60%, but it made OpenGL on NVIDIA *15 times* faster, making it now slightly faster than Vulkan at decoding ASTC textures.
+
+{{< imgs
+	"./astc.png| Subtle jump"
+  >}}
+
+GPUs which were left behind on Vulkan support can now enjoy much smoother performance in titles like `The Legend of Zelda: Tears of the Kingdom`, `Bayonetta 3`, `Luigi’s Mansion 3`, etc.
+
+To get the most out of this change, the `ASTC recompression` method must be set to `Uncompressed`, and `Enable asynchronous presentation` should be enabled if you’re using Vulkan,.
+Both options can be found in in `Emulation > Configure… > Graphics > Advanced`.
+
+[toastUnlimited]() later {{< gh-hovercard "11216" "blocked Mesa’s native ASTC decoder," >}} as it is now considerably slower than our implementation.
+
+And lastly, Epicboy solved what was a very common issue for `Pokémon Legends: Arceus` players before we made Vulkan the default API.
+Sit down, kids, it’s story time.
+
+In the not-so-far-away good old days, yuzu was only experimenting with an incomplete and immature Vulkan backend, there was no LDN/LAN support, and an Android release was considered to be a fever dream.
+Back then we used the OpenGL API by default, and out of its three options for shader backends, GLASM was the shader backend of choice, since it didn’t affect incompatible drivers–it’s an NVIDIA-only "feature", and any other driver would automatically revert to the vendor-agnostic GLSL. 
+This provided NVIDIA users the best experience at the time, with the lowest shader build times available, until Vulkan improved enough to be the full replacement it is today.
```

Comment:
```suggestion
Back then we used the OpenGL API by default, and out of its three options for shader backends, GLASM was the shader backend of choice, since it didn’t affect incompatible drivers — it’s an NVIDIA-only "feature" and any other driver would automatically revert to the vendor-agnostic GLSL. 
This provided NVIDIA users the best experience at the time, with the lowest shader build times available, until Vulkan improved enough to be the full replacement it is today.
```